src/compile/data.ts

```diff
@@ -287,13 +298,11 @@ export namespace formatParse {
     // if there is no parse needed
     model.forEach(function(fieldDef: FieldDef) {
       if (fieldDef.type === TEMPORAL) {
-        parseComponent = parseComponent || {};
```

Comment:
If you look above, you see that there is a `let parseComponent: Dict<string> = {};` so this is fine.